src/components/DatePickerContainer.ts

```diff
@@ -55,12 +55,12 @@ export default class DatePickerContainer extends Component<ContainerProps, Conta
     render() {
         const { mxObject } = this.props;
         const readOnly = this.props.editable === false
```

Comment:
i suggest a change to `const readOnly = !(this.props.editable)`

Reply:
Done